packages/block-library/src/read-more/index.php

```diff
@@ -19,15 +19,22 @@ function render_block_core_read_more( $attributes, $content, $block ) {
 	}
 
 	$post_ID            = $block->context['postId'];
+	$post_title         = get_the_title( $post_ID );
+	/* translators: %s is either the post title or post ID to describe the link for screen readers. */
+	$screen_reader_text = sprintf(
+		__( ': %s' ),
+		$post_title !== '' ? $post_title : __( 'post ' ) . $post_ID
```

Comment:
What's the use case where the post has no title? The post ID does differentiate links, but not in any way that's useful to the user, so I'm wondering if there's a better fallback option.

Reply:
Hmm I'm not sure. We could use `aria-describedby` in this case, but again not sure if this is any more valuable because it's linking it to post that has no title.

Would it be better to state that there is no post title, e.g. `Read More: (post with no title)`?

Reply:
In the scenario where an implementation uses primarily posts with no titles (ugly, but I've seen it), that would result in a series of identical links. Perhaps a combination, e.g. `Read More: untitled post $post_ID`

Reply:
> Perhaps a combination, e.g. Read More: untitled post $post_ID

Done in https://github.com/WordPress/gutenberg/pull/45490/commits/369691d870a4b20820dfc8b9270587222664a9b9